Topic6/src/main/java/com/globant/app/UserController.java

```diff
@@ -0,0 +1,100 @@
+package com.globant.app;
+
+import java.util.Collection;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.globant.model.User;
+import com.globant.services.UserService;
+import com.globant.services.UserServiceImpl;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+	
+	private UserService userService = new UserServiceImpl();
+
+	@RequestMapping("/getUsers")
+	@ApiOperation(value = "Finds all users", nickname = "Get all users")
+	@ApiResponses(value = { @ApiResponse(code = 200, message = "Success") })
+	public Collection<User> getUsers() {
+		return userService.getUsers();
+	}
+	
+	@RequestMapping("/findByRealname")
+	@ApiOperation(value = "Find users by real name", nickname = "Users by realname")
+	@ApiResponses(value = { @ApiResponse(code = 200, message = "Success"),
+			@ApiResponse(code = 404, message = "Not Found"), })

```

Comment:
the error code documented is not corresponding with error status returned by the service